src/main/java/io/github/oliviercailloux/y2018/apartments/valuefunction/profile/QuestionPriceArea.java

```diff
@@ -0,0 +1,109 @@
+package io.github.oliviercailloux.y2018.apartments.valuefunction.profile;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+import com.google.common.collect.Range;
+import io.github.oliviercailloux.y2018.apartments.valuefunction.Criterion;
+
+public class QuestionPriceArea {
+  int price;
+  int surface;
+  private final String question = "Would you pay %d€ more for %d m2 more?";
+
+  /**
+   * Instantiates a new question price/area.
+   *
+   * @param price the price shown to the user according to his/her profile
+   * @param surface the surface shown to the user according to his/her profile
+   */
+  private QuestionPriceArea(int price, int surface) {
+    checkNotNull(price);
+    checkNotNull(surface);
```

Comment:
Idem

Reply:
Fix in 5cb20dc6dd6f860db028a2a910b1575f02bdd942